phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1411,8 +1431,6 @@ public List<Mutation> getMutationList() {
                     child.stop();
                     child.stop();
                     shouldRetry = false;
-                    mutationCommitTime = EnvironmentEdgeManager.currentTimeMillis() - startTime;
-                    GLOBAL_MUTATION_COMMIT_TIME.update(mutationCommitTime);
```

Comment:
Nice. After [PHOENIX-6397](https://issues.apache.org/jira/browse/PHOENIX-6397) looks like we were updating this twice (same logic in `finally` as well).